app/views/measure_conditions/_measure_condition_code_document.html.erb

```diff
@@ -13,7 +13,7 @@
       <% if condition.document_code.present? %>
         <td class="govuk-table__cell numerical"><%= condition.document_code %></td>
         <td class="govuk-table__cell">
-          <%= condition.requirement %>
+          <%= condition.certificate_description.presence || condition.requirement %>
```

Comment:
I guess it need `.to_s.html_safe` as in `_measure_condition_code_default.html.erb `

Reply:
the html content in `requirement` appears to be for non-document measure conditions - and in that scenario certificate_description will be nil and fall back to `requirement` - `requirement` is already marked as html_safe in the data model.